### PR TITLE
Align structs name to tags

### LINKED ISSUE #261: Struct tags should not have different values than their field name
There are a few field names that differ from their struct tag representation. This is a bad practice as it often causes confusion and is also inconsistent. They should be aligned. An example:

https://github.com/observatorium/configuration/blob/c835853933cae03341a19fef1ce8a960b54a2707/api/v1alpha1/observatorium_types.go#L26

@nmagnezi @rollandf

### PR DESCRIPTION
**Pull request by rollandf**
Fixes #261

### CODE REVIEW

**Comment by rollandf** [comment]
@brancz @metalmatze @kakkoyun @squat PTAL

File: api/v1alpha1/observatorium_types.go [hunk 1]
```diff
@@ -23,17 +23,17 @@ import (
 // ObservatoriumSpec defines the desired state of Observatorium
 type ObservatoriumSpec struct {
 	// Thanos Image name
-	Image string `json:"thanosImage"`
+	ThanosImage string `json:"thanosImage"`
 	// Thanos Image version
-	Version string `json:"thanosVersion"`
+	ThanosVersion string `json:"thanosVersion"`
```
Comment (brancz) [inline api/v1alpha1/observatorium_types.go:28]: this should be under each component, not one version for all, versions should be individually selectable

Comment (brancz) [inline api/v1alpha1/observatorium_types.go:28]: this should be under each component, not one version for all, versions should be individually selectable

Comment (rollandf) [reply api/v1alpha1/observatorium_types.go:28]: This was fixed in another PR

Comment (brancz) [reply api/v1alpha1/observatorium_types.go:28]: sorry didn't see that yet, commented there

**Comment by rollandf** [comment]
I will consolidate in the PR #264

**Pull request closed without merging**